parsing/dialects/dialect1/config.fmpp

```diff
@@ -21,6 +21,8 @@ data: {
 
     # List of import statements.
     imports: [
+      "org.apache.calcite.sql.SqlJsonTypeNameSpec",
```

Comment:
nit: Let's put in alphabetical order. Actually, can you reorder existing imports in this file alphabetically as well while you're editing this?